### DISCUSSION TITLE
frontend

### DISCUSSION THREAD

**Discussion by Pavanyandrapati** [learndevopsonline/learndevopsonline#832] [Q&A]
Hi, i'm having this issue how to resolve..
![Screenshot 2023-05-11 185346](https://github.com/learndevopsonline/learndevopsonline/assets/129669919/7c9fe261-be75-448d-8611-7733d1a327ec)

**Comment by Arafique458** [comment]
I think you need define the file being copied. It shows as Template: I believe it should be copy. If you want to use template you need to change the file folder name to reflect as such.
